docs/content/preview/reference/drivers/csharp/yb-npgsql-reference.md

```diff
@@ -0,0 +1,201 @@
+---
+title: C# Drivers
+linkTitle: C# Drivers
+description: C# Drivers for YSQL
+headcontent: C# Drivers for YSQL
+image: /images/section_icons/sample-data/s_s1-sampledata-3x.png
+menu:
+  preview:
+    name: C# Drivers
+    identifier: ref-yb-npgsql-driver
+    parent: drivers
+    weight: 600
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../yb-npgsql-reference/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YugabyteDB Npgsql Smart Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="../postgres-npgsql-reference/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PostgreSQL Npgsql Driver
+    </a>
+  </li>
+
+</ul>
+
+The [Yugabyte Npgsql smart driver](https://github.com/yugabyte/npgsql) is a distributed .NET driver for [YSQL](../../../api/ysql/) built on the [PostgreSQL Npgsql driver](https://github.com/npgsql/npgsql/tree/main/src/Npgsql), with additional [connection load balancing](../../smart-drivers/) features.
+
+## Fundamentals
```

Comment:
Copy Go page format for driver dependency/Fundamentals sections; ie, driver dependency is its own top level section, followed by fundamentals